microsoft-edge/devtools-guide-chromium/accessibility/color-picker.md

```diff
@@ -0,0 +1,82 @@
+
+Contrast ratio measures the difference in brightness between the foreground and background of text.  If your text has a low contrast ratio, then these low vision users may literally experience your site as a blank screen.  
+
+In DevTools, one way to view the contrast ratio of a text element is to use the Color Picker, in the DOM tree in the **Elements** tool.  The Color Picker helps you verify that your text meets recommended contrast ratio levels.
+
+**To check the text-color contrast using the Color Picker:**
+
+1.  In DevTools, select the **Elements** tool.  
+1.  In the **DOM Tree**, select the text element that you want to inspect.  
+    
+    :::image type="complex" source="../media/accessibility-elements-paragraph-highlight.msft.png" alt-text="Inspect a paragraph in the DOM Tree" lightbox="../media/accessibility-elements-paragraph-highlight.msft.png":::
+       Inspect a paragraph in the **DOM Tree**  
+    :::image-end:::  
+    
+1.  On the **Styles** tab, locate the **color** property that's applied to the element, and then select the color square next to the **color** property.
+    
+    :::image type="complex" source="../media/accessibility-elements-styles-paragraph-highlight-color.msft.png" alt-text="The color property of the element" lightbox="../media/accessibility-elements-styles-paragraph-highlight-color.msft.png":::
+       The `color` property of the element  
+    :::image-end:::  
+    
+1.  Examine the **Contrast Ratio** section of the Color Picker.  One checkmark means that the element meets the [minimum recommendation][W3CContrastMinimum].  Two checkmarks means that it meets the [enhanced recommendation][W3CContrastEnhanced].  
+    
+    :::image type="complex" source="../media/accessibility-elements-styles-paragraph-highlight-color-picker.msft.png" alt-text="The Contrast Ratio section of the Color Picker shows 2 checkmarks and a value of 13.97" lightbox="../media/accessibility-elements-styles-paragraph-highlight-color-picker.msft.png":::
+       The **Contrast Ratio** section of the Color Picker shows 2 checkmarks and a value of `13.97`  
+    :::image-end:::  
+    
+1.  For more information, select the **Contrast ratio** section to expand it.  In the visual picker at the top of the Color Picker, two lines appear, running across the visual picker, along with a circle for the current color.  If the current color meets recommendations, then anything on the same side of the line also meets recommendations.  If the current color does not meet recommendations, then anything on the same side also does not meet recommendations.  
+
+    :::image type="complex" source="../media/accessibility-elements-styles-paragraph-highlight-color-picker-contrast-ratio-details.msft.png" alt-text="The Contrast Ratio Line in the visual picker" lightbox="../media/accessibility-elements-styles-paragraph-highlight-color-picker-contrast-ratio-details.msft.png":::
```

Comment:
I got mixed up with this image and the previous one. Can we add redlines around the section at the top of the color picker with the two curved lines and the circle?